src/test/java/core/basesyntax/service/impl/ReportGeneratorTest.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.storage.Storage;
+import org.junit.jupiter.api.Test;
+
+public class ReportGeneratorTest {
+    private ReportGenerator reportGenerator = new ReportGenerator();
+
+    @Test
+    void generateReport_validData_Ok() {
+        String actual = reportGenerator.generateReport(Storage.fruits);
+        assertEquals(String.class, actual.getClass());
+    }
```

Comment:
add a case with a non-empty storage, because this one does not really test a valid data